contracts/near/eth-client/src/tests.rs

```diff
@@ -461,3 +461,599 @@ fn predumped_block_can_be_added() {
     }
     bar.finish();
 }
+
+const DAGS_MERKLE_ROOT: &'static [&str] = &[
```

Comment:
This may be a good use for `hex_literal` crate.

Reply:
moved data to separate json file